addons/elasticsearch-curator/5.8.x/elasticsearch-curator-1.yaml

```diff
@@ -0,0 +1,87 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: elasticsearch-curator
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: elasticsearch-curator
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "5.8.1-1"
+    appversion.kubeaddons.mesosphere.io/elasticsearch-curator: "5.8.1"
+    values.chart.helm.kubeaddons.mesosphere.io/elasticsearch-curator: "https://raw.githubusercontent.com/helm/charts/139ee40/stable/elasticsearch-curator/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: gcp
+      enabled: true
+    - name: docker
+      enabled: false
+    - name: none
+      enabled: true
+  requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: elasticsearch
+  chartReference:
+    chart: stable/elasticsearch-curator
+    version: 2.1.5
+    values: |
+      ---
+      image:
+        tag: 5.8.1
+
+      resources: {}
+
+      cronjob:
+        # At 12:00 pm every day
+        schedule: "0 0 * * *"
+        annotations: {}
+        labels: {}
+        concurrencyPolicy: ""
+        failedJobsHistoryLimit: ""
+        successfulJobsHistoryLimit: ""
+        jobRestartPolicy: Never
+
+      pod:
+        annotations: {}
+        labels: {}
+
+      rbac:
+        enabled: true
+
+      configMaps:
+        # Delete indices older than 10 days
+        action_file_yml: |-
+          ---
+          actions:
+            1:
+              action: delete_indices
+              description: "Clean up ES by deleting indices older than 10 days"
+              options:
+                timeout_override:
```

Comment:
do we need this key if there's no value?

Reply:
> Looks good with 1 question. Also please add a release-note about changing the default time for deleting indices down from 30 to 10 days.

I haven't changed that value, just updated the description. It was already 10days.
See https://github.com/mesosphere/kubernetes-base-addons/pull/374/commits/69311e2bf0279570829750ee0ffb2503e73e4f8c

Reply:
oops you're right. nvm me.